content/docs/standard-library/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Standard Library
+weight: 5
+---
+This is the standard library section. Here you can find a collection of component descriptions and examples.
+These standard component's are minimal but useful and built on top of the primitive type's. 
+
+The neva standard library comes built in for all neva projects not requ``iring importing.
```

Comment:
1. `requ``iring`
2. "The neva standard library comes built in for all neva projects not requ``iring importing." - this is not really correct. Stdlib really does comes with compiler distribution but you have to import all packages except `builtin`. This is explained in tutorial.